src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -94,6 +94,19 @@ public enum UnreferencedTypesHandlingOption
             {
                 _absoluteProjectFolder = Path.GetFullPath(Path.Combine(outputFolder, projectFolder));
             }
+
+            if (publicClients && generation1ConvenienceClient)
+            {
+                //TODO Remove after resolving https://github.com/Azure/autorest.csharp/issues/3151
+                if (_absoluteProjectFolder is not null &&
```

Comment:
Should we consider adding slugs around the values, for example:  `/Azure.Communication.PhoneNumbers/`?    Otherwise, wouldn't we potentially allow things like `Azure.Communication.PhoneNumbers.Blah`?